<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bachelorClass</class>
 <widget class="QMainWindow" name="bachelorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bachelor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>641</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>edit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bachelor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>bachelorClass</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>813</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget_2</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>bachelorClass</receiver>
   <slot>addStudent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>846</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>bachelorClass</receiver>
   <slot>editStudentInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search()</slot>
  <slot>addStudent()</slot>
  <slot>editStudentInfo()</slot>
 </slots>
</ui>
